spec/services/account_reset/cancel_request_for_user_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe AccountReset::CancelRequestForUser do
+  let(:user) { create(:user) }
+  let!(:account_reset_request) { AccountResetRequest.create(user: user, requested_at: 1.hour.ago) }
+
+  subject { described_class.new(user) }
+
+  describe '#call' do
+    it 'cancels the account reset request' do
+      subject.call
+
+      expect(account_reset_request.reload.cancelled_at).to be_within(1.second).of(Time.zone.now)
```

Comment:
so now that `.call` has a default parameter of `now`, we can do:

```
let(:now) { Time.zone.now }

it 'cancels the account reset request' do
  subject.call(now: now)

  expect(account_reset_request.reload.cancelled_at.to_i).to eq(now.to_i)
end
```

Out of habit I tend to convert timestamps to integers like this when round-tripping from the DB, but it might work without here